src/model/webhook.rs

```diff
@@ -234,6 +234,15 @@ impl Webhook {
             Err(why) => Err(why),
         }
     }
+
+    /// Returns the url of the webhook.
+    ///
+    /// ```rust,no_run
+    /// assert_eq!(hook.url(), "https://discord.com/api/webhooks/245037420704169985/ig5AO-wdVWpCBtUUMxmgsWryqgsW3DChbKYOINftJ4DCrUbnkedoYZD0VOH1QLr-S3sV".to_string())
```

Comment:
```suggestion
    /// assert_eq!(hook.url(), "https://discord.com/api/webhooks/245037420704169985/ig5AO-wdVWpCBtUUMxmgsWryqgsW3DChbKYOINftJ4DCrUbnkedoYZD0VOH1QLr-S3sV")
```
[`String` implements equality comparisons to `&str`](https://doc.rust-lang.org/nightly/std/string/struct.String.html#impl-PartialEq%3C%26%27a%20str%3E), you don't need to elevate the literal to a `String` before being able to use it in the `assert_eq!` macro.